Gemfile.lock

```diff
@@ -1,60 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.5.0)
+    better_errors (2.5.1)
       coderay (>= 1.0.0)
       erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (10.0.2)
+    byebug (11.0.1)
```

Comment:
### [byebug](https://github.com/deivid-rodriguez/byebug)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 10.0.2 → 11.0.1

[_~~change-log~~, [source-code](https://github.com/deivid-rodriguez/byebug)_]





<details>
<summary>Commits</summary>

A change of **204** commits. See the full changes on [the compare page](https://github.com/deivid-rodriguez/byebug/compare/v10.0.2...v11.0.1).

These are the first 10 commits:
- (1ce507d) [Bump rubocop from 0.54.0 to 0.55.0 (#454)](https://github.com/deivid-rodriguez/byebug/commit/1ce507dbde83ec245c00166db070d32563a19ab2)
- (0849994) [Appveyor & Travis - Bundler 1.16.2, Appveyor misc (#459)](https://github.com/deivid-rodriguez/byebug/commit/0849994f4dda4f73bdb3285cccbdd13e63204c65)
- (ac7197e) [Stop explicitly installing bundler](https://github.com/deivid-rodriguez/byebug/commit/ac7197ebd5c0bdbffe57973a8256004544a4a185)
- (b1254b8) [Bump rubygems to 2.7.7](https://github.com/deivid-rodriguez/byebug/commit/b1254b8db3d3d49d2b6ac0797c35c49c7441868f)
- (e362d60) [Allow upgrading locked bundler version](https://github.com/deivid-rodriguez/byebug/commit/e362d60224bc7a4dfddc6d5e19274776941c2e26)
- (ca5037c) [bundle update --bundler](https://github.com/deivid-rodriguez/byebug/commit/ca5037c611e6cd1118b998a5b8f6f8ca299c9e3f)
- (297d540) [Silence docker:test output by default](https://github.com/deivid-rodriguez/byebug/commit/297d5403476be36dfcef9e32401c6fdc8a1208b0)
- (a7fe679) [bundle update unicode-display_width](https://github.com/deivid-rodriguez/byebug/commit/a7fe6793f937df6e2f5eec7e1d834baeb4be9adb)
- (93be833) [Bump test reporter to 0.6.0](https://github.com/deivid-rodriguez/byebug/commit/93be83378bfef86c94bf3065846f3b2eed1d2358)
- (69c58a4) [Add missing checkout to upload coverage step](https://github.com/deivid-rodriguez/byebug/commit/69c58a4118c78aebe3488cd4d5146440cf3f60ca)


</details>